ydb/core/persqueue/partition.h

```diff
@@ -795,6 +804,19 @@ class TPartition : public TActorBootstrapped<TPartition> {
     bool ClosedInternalPartition = false;
 
     bool IsSupportive() const;
+
+    ui64 DeletePartitionCookie = 0;
```

Comment:
Change to ENUM:
DELETEION_NOT_INITED
DELETION_INITED
DELETION_IN_PROCESS

Reply:
Done
https://github.com/ydb-platform/ydb/pull/4407/commits/76c2d74fad557fb217fc4d91fc813ada28978cf8